source/algorithms.tex

```diff
@@ -8773,14 +8779,14 @@
 
 \pnum
 \returns
-The smallest value in the input range.
-Returns a copy of the leftmost element
-when several elements are equivalent to the smallest.
+Returns a copy of the leftmost element \tcode{e} for which
```

Comment:
Should this (and below) say "...the leftmost element \tcode{e} <ins>in the input range \tcode{r}</ins> for which ... in <del>the input range</del> \tcode{r}"?

That is, the words "the input range" appear late here, and don't appear in lines 8866–8868 at all; and we aren't yet saying "element `e`" of _what_, exactly.

Reply:
LGTM now!